demo/docroot/.ht.router.php

```diff
@@ -26,8 +26,15 @@
 
 $url = parse_url($_SERVER['REQUEST_URI']);
 if (file_exists(__DIR__ . $url['path'])) {
-  // Serve the requested resource as-is.
-  return FALSE;
+
+  if (strpos($url['path'], '.js') !==  FALSE) {
+    header('Access-Control-Request-Method: GET');
+    header('Access-Control-Allow-Origin: *');
```

Comment:
Could we have the specific domain defined in `settings.php` instead of setting it to `*`?

Reply:
Which entry in settings.php are you referrring to? Note: This is just for development on the admin UI  itself